driver/src/main/java/oracle/nosql/driver/http/Client.java

```diff
@@ -1037,6 +1051,36 @@ private void processNotOKResponse(HttpResponseStatus status,
                                  ", reason = " + status.reasonPhrase());
     }
 
+    /* set session cookie, if set in response headers */
+    private void setSessionCookie(HttpHeaders headers) {
+        if (headers == null) {
+            return;
+        }
+        List<String> values = headers.getAll("Set-Cookie");
```

Comment:
I think you can get by with the simple "get" for this which simplifies the logic here

Reply:
If there's more than one Set-Cookie header (not likely, but _possible_), then get() only gets the first one. Hence this code, to be sure.

Reply:
I don't think that multiple Set-Cookie headers is legitimate, at least in our path. getAll() creates a new List for results and get() does not (from my reading of Netty)

Reply:
OK, I'll change it and add a comment. If the LB ever does try to set two cookies then this won't work.